infrastructure/main.tf

```diff
@@ -41,6 +41,7 @@ module "citizen-frontend" {
   is_frontend  = true
   subscription = "${var.subscription}"
   additional_host_name = "${var.external_host_name}"
+  https_only = "true"
```

Comment:
Would `https_only = true` work? (boolean instead of string)

Reply:
nope. module is inconsistent github.com/hmcts/moj-module-webapp README

Reply:
Thanks for the link

Changing subject, I just realised we removed HPKP on CNP, something we need to enable in public BETA. Not necessarily first days.

Reply:
Yeah lets see whether Google sticks to their plan. I think so far only they made that move.